### DISCUSSION TITLE
awitching or adopting psycopg3 aync version fro sqlalchmy 2.0? or may be 1.5?

### DISCUSSION THREAD

**Discussion by auvipy** [sqlalchemy/sqlalchemy#7208] [Ideas]
psycopg3 is a async python version of psycopg2
https://www.psycopg.org/psycopg3/docs/.

should we consider adapting this? may be this could be helpful for async sqlalchemy postgres? I would like to know the mindset of the core teams, if need might try a PoC PR.

**Comment by CaselIT** [comment]
Hi,

It's been worked on here https://github.com/sqlalchemy/sqlalchemy/issues/6842

**Comment by item4** [comment]
any update more? I want to know why psycopg3 support do not included in 1.x version. It seems to merged only 2.x branch. There is no plan for psycopg3 support with 1.4 or higher minor version?

**Reply by auvipy** [reply]
it is natural to include newer versions of dependencies on new major releases.

**Reply by CaselIT** [reply]
replied here https://github.com/sqlalchemy/sqlalchemy/issues/6842#issuecomment-995211004
